.gitignore

```diff
@@ -56,7 +56,7 @@ coverage.xml
 *.pot
 
 # Django stuff:
-*.log
+*.log*
```

Comment:
This would only ignore for this repository. The language plugins have `.gitignore` files used for repos created with them where we should probably add this:

[Python](https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/blob/master/python/rpdk/python/data/Python.gitignore)
[Go](https://github.com/aws-cloudformation/cloudformation-cli-go-plugin/blob/master/python/rpdk/go/data/go.gitignore)
[Java](https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/python/rpdk/java/data/java.gitignore)